build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+/*
+ * SonarSource Go
+ * Copyright (C) 2018-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the Sonar Source-Available License Version 1, as published by SonarSource SA.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the Sonar Source-Available License for more details.
+ *
+ * You should have received a copy of the Sonar Source-Available License
+ * along with this program; if not, see https://sonarsource.com/license/ssal/
+ */
+plugins {
+    id("org.sonarsource.cloud-native.code-style-conventions")
+    id("org.sonarsource.cloud-native.artifactory-configuration")
+    id("org.sonarqube") version "6.0.1.5171"
+}
+
+artifactoryConfiguration {
```

Comment:
This is also configured in sonar-go-plugin/build.gradle.kts. Can it be in one place only?

Reply:
IMO it's better to configure publication in the same project, that is published. However, the Jfrog plugin needs certain configuration on the root project specifically. I don't think it should be all in one place.

Reply:
Split `artifactory-configuration` into `artifactory-configuration` (applied to the root project only) and `publishing-configuration` (applied to the plugin)